packages/mobile/src/navigator/NavigatorWrapper.tsx

```diff
@@ -150,6 +151,11 @@ export const NavigatorWrapper = () => {
         currentScreen: currentRouteName,
       })
       dispatch(activeScreenChanged(currentRouteName))
+      Sentry.addBreadcrumb({
+        category: 'navigation',
+        message: `Navigated to ${currentRouteName}`,
+        level: Sentry.Severity.Info,
+      })
```

Comment:
was wondering if these breadcrumbs provide different info to the `sentryRoutingInstrumentation` which tracks the navigation already? i tried looking at the error events/navigation events of the last few days but i couldn't see any extra breadcrumbs in Sentry. if you could point me to an example i'm keen to see what the breadcrumbs look like!

Reply:
🤔 the traces sample rate is 0.2 i think, so is it possible that the first event was sampled and the second event wasn't? i see this event from 1.24.1 that has a navigation breadcrumb already https://sentry.io/organizations/valora-inc/performance/celo-mobile:055880cf850442238b44cd855d127305/?environment=co.clabs.valora&project=1250733&query=transaction.duration%3A%3C15m+event.type%3Atransaction&statsPeriod=14d&transaction=WalletHome&unselectedSeries=p100%28%29 which looks pretty similar to the breadcrumb provided by this snippet...

Reply:
having said that, i'm not sure if the react navigation tracing leaves breadcrumbs on error events. that event i linked is from the performances trace, so may be different. i'm having a hard time finding the navigation breadcrumb on the recent errors, so not really sure what's going on. in any case, i am happy to keep this block here!

Reply:
I'll try a couple more times to see how consistent it is. I would expect the tracing limit to affect performance stuff but not errors! If there's an error I want to see the breadcrumbs all the time, not just 20% of the time :P I'll test and report back

Reply:
Yeah, I tested 4 more times, the two times that I added the breadcrumb both were there: https://sentry.io/organizations/valora-inc/issues/2940380905/?project=1250733&query=is%3Aunresolved and https://sentry.io/organizations/valora-inc/issues/2940386168/?project=1250733&query=is%3Aunresolved and the two times I didn't no navigation breadcrumb showed up: https://sentry.io/organizations/valora-inc/issues/2940386748/?project=1250733&query=is%3Aunresolved and https://sentry.io/organizations/valora-inc/issues/2940387891/?project=1250733&query=is%3Aunresolved